src/utils/formatNumber.ts

```diff
@@ -113,11 +119,11 @@ export const formattedNum = (
     if (post === '0') return formatNearZero(pre)
 
     const formattedPost = post
-      .substr(0, config?.decimals ?? 18)
-      .padEnd(config?.padEnd ? config?.decimals ?? 0 : 0, '0')
+      .substring(0, config?.precision ?? 18)
```

Comment:
**question (non-blocking)**: is falling back to a precision of `18` meaningfully related to `WAD_DECIMALS` (which is also `18`)?

Reply:
definitely meaningfully related! it's probably better to use `WAD_DECIMALS` here instead `18`.